include/s3select_oper.h

```diff
@@ -567,6 +573,14 @@ class value
     return *this;
   }
 
+  value& operator=(bool b)
+  {
```

Comment:
not correct, its override existing functionality. 
who calls this operations?

what should be done , when push_null is called , a variable should be set to null(its value), explicitly.